### DISCUSSION TITLE
/stations/{stationId}/tafs unexpectedly returns an empty graph

### DISCUSSION THREAD

**Discussion by 1npo** [weather-gov/api#759] [General]
I'm not getting any data back from the `/stations/KBOX/tafs` endpoint, but I know there's a terminal aerodrome forecast for KBOX. (I've also tried this for KBOS and about a dozen other station IDs at or near airports.) Is there an issue with the endpoint, or is the data no longer available from the API?

### KBOX TAF:
![image](https://github.com/user-attachments/assets/8a608d9e-fa68-4a93-82cd-6dbac4718e31)

### Request made from Swagger UI on weather.gov
![image](https://github.com/user-attachments/assets/ca70f693-6cac-42c2-811c-322777667161)

### Request made from API client:
![image](https://github.com/user-attachments/assets/2826141a-f6c5-4033-8cda-817854e2f103)

**Comment by dwhitemv25** [comment]
Tangent -- TAFs are also available through the [aviationweather.gov API](https://aviationweather.gov/data/api/).

**Reply by 1npo** [reply]
TIL about that API - thank you! Looking at it, I wouldn't be surprised if the `/stations/{stationId}/tafs` and even `/aviation/*` endpoints are being deprecated in favor of the aviationweather API.

**Reply by sullynole** [reply]
TAFBOS is also available in API here: https://api.weather.gov/products/types/TAF/locations/BOS
